crates/database/src/orders.rs

```diff
@@ -651,29 +671,44 @@ pub struct FeeUpdate {
     pub surplus_fee: Option<BigDecimal>,
     pub surplus_fee_timestamp: DateTime<Utc>,
     pub full_fee_amount: BigDecimal,
+    pub sell_token: Address,
+    pub buy_token: Address,
+    pub sell_amount: BigDecimal,
 }
 
+/// Updates the `surplus_fee` of multiple orders and returns their `uid`s.
 pub async fn update_limit_order_fees(
     ex: &mut PgConnection,
-    order_uid: &OrderUid,
     update: &FeeUpdate,
```

Comment:
uber-nit: Feel free to ignore.

Maybe it makes sense to split `FeeUpdate` into:
```rust
struct OrderFeeSpecifier {
    pub sell_token: Address,
    pub buy_token: Address,
    pub sell_amount: BigDecimal,
}

struct FeeUpdate {
    pub surplus_fee: Option<BigDecimal>,
    pub surplus_fee_timestamp: DateTime<Utc>,
    pub full_fee_amount: BigDecimal,
}
```

This way we keep the same analogy:
```diff
-update_limit_order_fees(order_uid, fee_update).await?;
+update_limit_order_fees(order_spec, fee_update).await?;
```

This is just splitting hairs though, so feel free to ignore the comment.

Reply:
I already added a struct which had those exact fields but I didn't use it consistently in this PR. Using it everywhere like you suggested made the code a bit nicer. 👍 
Also I renamed the existing struct to your suggestion because it sounds nicer.
[f9ef7f0](https://github.com/cowprotocol/services/pull/1037/commits/f9ef7f0897db5d3d38976d947937304d0be3cccc)